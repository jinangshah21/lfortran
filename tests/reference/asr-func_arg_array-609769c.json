{
    "basename": "asr-func_arg_array-609769c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/func_arg_array.f90",
    "infile_hash": "f673ab43128424f50557ce06f2aa2424f8b1f119f84da7c07e8cd85d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-func_arg_array-609769c.stderr",
    "stderr_hash": "83deb3db6f5fffbc372738b46861eabe16baa6e50c0097b34103cdc7",
    "returncode": 2
}