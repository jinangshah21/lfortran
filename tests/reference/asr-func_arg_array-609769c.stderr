semantic error: Incompatible rank `2` array passed in function which expects rank `3`.
 --> tests/errors/func_arg_array.f90:4:14
  |
4 |     i = temp(myArray)
  |              ^^^^^^^ 
